src/dsymbol/conversion/package.d

```diff
@@ -193,18 +202,29 @@ class SimpleParser : Parser
 				if (moreTokens)
 					advance();
 				if (currentIs(tok!"("))
+				{
+					bool asExpr;
+					if (index < tokens.length - 2 &&
```

Comment:
and finally this just to be more correct whith the `needDo`, it's necessary to detect if the out contract is an expression style or not.